### PR TITLE
SPE-206 Init readme.md in models/_sector/dex/

### PR DESCRIPTION
**Pull request by Hosuke**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/_sector/dex/readme.md
Comment (Hosuke) [inline models/_sector/dex/readme.md:1]: Hi @aalan3 @jeff-dude, I am currently drafting the README for `models/_sector/dex/`. If there are any additional points, suggestions, or specific information you believe should be included, please let me know.

I will continue to refine and expand the document.

Thanks in advance for your help!

Comment (jeff-dude) [reply models/_sector/dex/readme.md:1]: i'm going to comment items as i come across them in reviewing open PRs, so bare with me as comments come in over time 🙏 

first thought:
- we should include the dependency on `dex.info` and how to add a row there when adding a new dex -- be sure to include the why as well

File: models/_sector/dex/readme.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+## Macro vs. Code in Model
+
+- Standalone dex: Code directly in model is acceptable.
+
+- Forked dex/Repeatable logic: Use or create a macro. eg.
+```sql
+{{
+    uniswap_compatible_v2_trades(
```
Comment (jeff-dude) [inline models/_sector/dex/readme.md:55]: maybe link to this macro, for easy navigation?

File: models/_sector/dex/readme.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        version = '2',
+        Pair_evt_Swap = source('biswap_bnb', 'BiswapPair_evt_Swap'),
+        Factory_evt_PairCreated = source('biswap_bnb', 'BiswapFactory_evt_PairCreated')
+    )
+}}
+```
+
+## Further Information
+
+For more details, please refer to relevant GitHub issues, PRs, and comments. This repository is a collective effort, and your contributions are highly valued.
```
Comment (jeff-dude) [inline models/_sector/dex/readme.md:67]: can you link the relevant gh issues / initial PR that had structure/uniswap?

File: models/_sector/dex/readme.md
Comment (jeff-dude) [reply models/_sector/dex/readme.md:1]: we should consider a section on adding seed tests, how to add, & why we add it (test-driven development, to prove output of model matches expected results hardcoded in seed)

File: models/_sector/dex/readme.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# README for models/_sector/dex/
+
+Welcome to the `models/_sector/dex/` directory of our project. This document provides essential information on contributing to the dex sector models, along with insights into our design choices and the next steps in the dex.trades redesign workstream.
+
+## How to Contribute
+
+1. **Understanding the Structure**: Familiarize yourself with the current structure and conventions used in the dex sector models.
```
Comment (jeff-dude) [inline models/_sector/dex/readme.md:7]: we should add more details on the structure and why. for example, issue #4637 description, specifically the `what's changing (and why)?` section that talks about project spells being raw/decoded only, how we union projects within a blockchain, adding metadata at the end (and why we wait until the end)

**Review by jeff-dude — APPROVED** [review]
let's get this merged in & we can modify as we continue. thanks for initiating!

**Pull request merged**